Recruitment/JobSpecs/why_you_js_engineer.md

```diff
@@ -0,0 +1,43 @@
+### Why You?
+
+Our clients know that there's a better way to deliver software, but aren't always sure what it is, or how to get there. When they are ready to change, we help them.
+
+If that sounds like something you'd like to be a part of, here are some things to consider:
+
+#### The Responsibilities:
+
+* Working with the Engineering, Business Strategy, UX/UI and Design teams as well as customer organizations to architect, design, and implement solutions.
+* Designing, developing and productising front-end components and services that complete our Digital Rig – a fully automated platform and continuous delivery stack we use to bootstrap and accelerate Customer Journey Engineering.
+* Determine, apply and improve appropriate methodologies and toolsets for a customer’s culture and operating environment.
+* Participate in and lead digital workshops with customer organizations.
+* Complete solutions by coordinating, communicating, and contributing to team meetings and customer engagements.
+* Explore up-and-coming technologies and software products; seize learning and knowledge sharing opportunities; maintain personal networks; participate in professional organizations. Read; teach; learn.
+* Enhance the group’s reputation by accepting ownership for accomplishing new and different requests; exploring opportunities to add value.
+* Support the team by providing advice, coaching, and educational opportunities.
+
+#### You:
+
+* Are talented and passionate Front-end Developer.
```

Comment:
Thanks! I think would be better to change this to Front-End Engineer, with a focus on JS Engineering? Having discussions about pushing JSE more than HTML/CSS.